lib/commands/exam.js

```diff
@@ -101,6 +116,47 @@ module.exports = TestCommand.extend({
     return this._super.run.apply(this, arguments);
   },
 
+  /**
+   *
+   * Instantiate an executionMappingObj object
+   *
+   * @param {Object} option
+   */
+  getExecutionMappigInstance: function(options) {
+    if (executionMappingObj !== undefined && executionMappingObj['numberOfBrowsers'] !== undefined && executionMappingObj['executionMapping'] !== undefined) {
+      return executionMappingObj;
+    }
+    const filePath = options.replayExecution;
+    const replayBrowser = options.replayBrowser;
+
+    let testMappingObj = {};
+
+    try {
+      // Read the replay execution json file.
+      const testMappingRawData = fs.readFileSync(filePath);
+      testMappingObj = JSON.parse(testMappingRawData);
+    } catch (err) {
+      const errorMsg = 'Error reading reply execution JSON file - ' + err;
+      throw new Error(errorMsg);
+    }
+
+    const browserModuleMappingObj = testMappingObj['executionMapping'];
+
+    executionMappingObj['numberOfBrowsers'] = testMappingObj['numberOfBrowsers'];
+    executionMappingObj['executionMapping'] = ((replayBrowser, browserModuleMappingObj) => {
+      const browserModuleMap = {};
+
+      for (const idx in replayBrowser) {
+        const browserId = replayBrowser[idx];
```

Comment:
I'm a bit confused by this, as `replayBrowser` is defined as this in the options:

> { name: 'replay-browser',   type: [Array, Number],          description: 'The number of the browser(s) to run from a specified file path'}

Is this either an `Array<int>` _or_ an `Array<Array<int>>`?

Reply:
`replayBrowser` is an `Array<int>`. in `for .. in` a value of idx is an index of replayBrowser. for instance, replayBrowser = [1, 2, 3] then in the first iteration of the for loop idx value will be 0 and browserId will be replayBrowser[0]. To reduce confusion here what about using `for( ; ; ;)` to iterate an array?